requirements.in

```diff
@@ -1,30 +1,199 @@
+absl-py==0.11.0
```

Comment:
This seems like it may be excessive but maybe we really are using this much lol.

Reply:
i think it was part of the sentence_embedding code i was using earlier. right now, i'm only importing pandas and tensorflow. however, i'm not sure how to delete it, do i just remove this line from the file, or do i have to run a specific command?

Reply:
I meant the entire file had a ton of libraries, but don't worry about it. Once we feel like we're done, I'll run everything on my machine and then update the libraries. You don't need to worry about it. You can just leave this.